src/main/webapp/login-script.js

```diff
@@ -67,3 +77,39 @@ function signOutPrivate() {
   document.getElementById("private-class-info").classList.add("hidden");
   document.getElementById("private-login-box").classList.remove("hidden");
 }
+
+async function getTermList() {
+  const school = getUserSchool();
+  const url = new URL("/term", window.location.origin);
+  url.searchParams.set("school-name", school);
+  const response = await fetch(url);
+  const json = await response.json();
+  return json.terms;
+}
+
+function getUserSchool() {
+  const auth2 = gapi.auth2.getAuthInstance();
+  const profile = auth2.currentUser.get().getBasicProfile();
+  const email = profile.getEmail();
+  const start = email.indexOf("@");
+  const end = email.lastIndexOf(".");
+  return email.substring(start + 1, end);
+}
+
+function createOptionElement(val) {
+  const opt = document.createElement('option');
+  opt.appendChild(document.createTextNode(val));
+  opt.value = val; 
+  return opt;
```

Comment:
Better naming again for val and opt, though not as critical here as the function name is good.